src/server/info_collector.cpp

```diff
@@ -150,15 +146,15 @@ void info_collector::on_app_stat()
         // get row data statistics for all of the apps
         all_stats.merge(app_stats);
 
-        // hotspot_calculator is to detect hotspots
-        hotspot_calculator *hotspot_calculator =
+        // hotspot_partition_calculator is to detect hotspots
+        hotspot_partition_calculator *hotspot_partition_calculator =
             get_hotspot_calculator(app_rows.first, app_rows.second.size());
-        if (!hotspot_calculator) {
+        if (!hotspot_partition_calculator) {
```

Comment:
A nullptr `hotspot_partition_calculator` is an invalid case, better to use `dassert` here.